src/lib/common/limits.h

```diff
@@ -99,34 +99,17 @@
 
 /* ****************************************************************************
 *
-* Message definitions -
+* DEFAULT_IN_REQ_PAYLOAD_MAX_SIZE -
 */
-#define MAX_STA_MSG_SIZE (20 * 1024)           // 20 KB (HTTP request static buffer)
-#define MAX_DYN_MSG_SIZE (8 * 1024 * 1024)     // 8 MB  (maximum length of the HTTP request dynamic buffer)
```

Comment:
As far as I checked, MAX_STA_MSG_SIZE and MAX_DYN_MSG_SIZE  were not used, so I have removed them.

NTC